recipes/mixsimulator/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "mixsimulator" %}
+{% set version = "0.2.9.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mixsimulator-{{ version }}.tar.gz
+  sha256: 0990c51202902c5e3a7bfa747b7e5869877738e21306d8e86763d7513214671c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - mixsimulator
+    - mixsimulator.centrals
+
+about:
+  home: https://github.com/Foloso/MixSimulator
+  summary: Python application with nevergrad optimization model for calculating and simulating the least cost of an energy Mix under constraints.
+  license_file: LICENSE.txt
+  license: MIT
+  license_family: MIT
+  doc_url: https://facebookresearch.github.io/nevergrad/
```

Comment:
This `doc_url` and `dev_url` look to be incorrect

Reply:
You're correct! Must have been a copy/paste error, the correct urls should be: https://github.com/Foloso/MixSimulator 

Should I do an update PR?

Reply:
You'll need to open a PR in your person fork of the feedstock (https://github.com/conda-forge/mixsimulator-feedstock). You can just increment the build number as well and rerender the feedstock to update the README and other metadata. Let me know if you need any assistance or to take a look at the PR.